src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -513,15 +514,13 @@ export class WorkspaceSavedObjectsClientWrapper {
            */
           options.workspaces = permittedWorkspaces;
         } else {
-          /**
-           * If no workspaces present, find all the docs that
-           * ACL matches read / write / user passed permission
-           */
-          options.ACLSearchParams.permissionModes = getDefaultValuesForEmpty(
-            options.ACLSearchParams.permissionModes,
-            [WorkspacePermissionMode.Read, WorkspacePermissionMode.Write]
-          );
-          options.ACLSearchParams.principals = principals;
```

Comment:
The code here is used for searching user object and could we keep it as it is? We can move the return block to line 460 maybe?

Reply:
Updated